examples/lines_bars_and_markers/barh_demo.py

```diff
@@ -2,9 +2,8 @@
 Simple demo of a horizontal bar chart.
 """
 import matplotlib.pyplot as plt
-plt.rcdefaults()
 import numpy as np
-import matplotlib.pyplot as plt
+plt.rcdefaults()
```

Comment:
If you're going to move this down here, it should have two blank lines above and one blank line after.